WordPress/Classes/ViewRelated/Blog/Blog + Me/GravatarButtonView.swift

```diff
@@ -66,13 +66,7 @@ extension GravatarButtonView {
 extension GravatarButtonView {
 
     private struct StandardBorder {
-        static var color: UIColor {
-            if #available(iOS 13, *) {
-                return .separator
-            }
-
-            return .gray(.shade20)
-        }
+        static var color: UIColor = .separator
```

Comment:
At that point I think it can be a `static let`?
```suggestion
        static let color: UIColor = .separator
```